rust-toolchain.toml

```diff
@@ -0,0 +1,3 @@
+[toolchain]
+channel = "nightly"
```

Comment:
Good call to introduce this, but we should be explicit about which nightly release to use